GiniVision/Classes/Core/Screens/Camera/Camera.swift

```diff
@@ -129,6 +129,11 @@ final class Camera: NSObject, CameraProtocol {
     }
     
     func captureStillImage(completion: @escaping (Data?, CameraError?) -> Void) {
+        
+        // Reuse settings for multiple captures
```

Comment:
Is this comment valid? I don't understand how the settings can be reused if we use a local variable instead of a class variable. Is the call back below reused then?

Reply:
According to https://developer.apple.com/documentation/avfoundation/cameras_and_media_capture/capturing_still_and_live_photos -
It is illegal to reuse a AVCapturePhotoSettings instance for multiple captures.To reuse settings for multiple captures, use init(from:).

Reply:
I've left the long comment in the commit message.

Reply:
My question was not about the change but if the comment is valid. I understand now that you use the same settings but with new object instances. It was not clear for me from the comment. Maybe change the comment to "Reuse settings with new instances" or "Reuse settings safely for multiple captures" or just delete it. This is up to you - I understand the context now better.